helmfile/values/prometheus-alerts-wc.yaml.gotmpl

```
@@ -17,3 +17,6 @@ defaultRules:
     rookMonitor: {{ .Values.monitoring.rook.enabled }}
 
 predictLinearLimit: {{ .Values.prometheus.predictLinearLimit }}
+
+s3BucketPercentLimit: {{ .Values.prometheus.s3BucketPercentLimit }}
+s3BucketMaxSizeLimit: {{ .Values.prometheus.s3BucketMaxSizeLimit }}
```

I think you need to add this, since the user alertmanager will not have access to these metrics.
```suggestion
    rookMonitor: {{ .Values.monitoring.rook.enabled }}
    backupStatus: false
    dailyChecks: false

predictLinearLimit: {{ .Values.prometheus.predictLinearLimit }}

s3BucketPercentLimit: {{ .Values.prometheus.s3BucketPercentLimit }}
s3BucketMaxSizeLimit: {{ .Values.prometheus.s3BucketMaxSizeLimit }}
```